processors/classes/ui.php

```diff
@@ -347,6 +350,21 @@ public static function input( $type, $args, $id, $classes, $required, $has_desc
 					'{{' . esc_attr( $id ) . '}}'
 				);
 				break;
+			case 'dynamic':
+				$field = sprintf('<select id="{{_name}}[%s]" class="%s dynamic" name="{{_name}}[%s]" data-value="{{%s}}" data-callback="%s"></select>',
```

Comment:
This addition makes the inline docs for this method inaccurate. Please update them as well.